mempool/reactor.go

```diff
@@ -103,36 +103,50 @@ func (memR *Reactor) GetChannels() []*p2p.ChannelDescriptor {
 // AddPeer implements Reactor.
 // It starts a broadcast routine ensuring all txs are forwarded to the given peer.
 func (memR *Reactor) AddPeer(peer p2p.Peer) {
+	ctxParent := context.TODO() // This context should come from the caller.
```

Comment:
Comment out?

Reply:
Done. Cleanup up the other comment as well.